README.md

```diff
@@ -24,6 +24,8 @@ When there are machines which have access to both the public internet and the GH
 
 **Arguments:**
 
+- `actions-admin-user` _(optional)_
+   The name of the Actions admin user, which will be used for updating the chosen action. If not specified `actions-admin` will be used.
```

Comment:
Just a basic question:
```
When creating a personal access token include the repo and workflow scopes. 
Include the site_admin scope (optional) if you want organizations to be created as necessary.
```

With this impersonation to work, what kind of access scope is needed ?

Reply:
When I tested this manually, I used the same scopes as I would normally would for the token: `public_repo` (or `repo` for the AE instance) and `workflow` and it worked for me.
So it seems like nothing is changing there.

Reply:
I tested it with GHES today and for whatever reason, I needed to have a `site_admin` scope in my token. I will investigate further.

Reply:
@chrisgavin should this also be true for the AE instances?

Reply:
AFAIK, yes.

Reply:
Thanks @derwasp , @chrisgavin for responses.


I think for GHAE as well, not sure why the behaviour is different, but i agree with @chrisgavin that GHAE instance also should have `site_admin` permissions
Otherwise, normal customers with less priviliges also be able to sync via impersonation ?

Reply:
@ajaykn 
Who can we verify this behavior with? Unless I am missing something, right now the GHAE instances clearly are not expecting the `site_admin` permission for the impersonation requests as I verified with the logs above.
I don't mind just asking for the `site_admin` permission in both cases, but I wanted to minimize the required permissions unless really necessary.

Reply:
@derwasp It seems, we are checking for site_admin even in codeql-sync tool.
https://github.com/github/codeql-action-sync-tool/blob/main/internal/push/push.go#L107

Can we do the same regardless of how it works in ghae.
Making sure `site_admin` permissions needed for impersonation for ghae as well.

And we can start conversation in slack channels like `actions-on-ghae` or `ghae-engineering` to see, if the impersonation logic working regardless of site_admin.